### PR TITLE
fix amp-ad-exit in singlepass

### PR DESCRIPTION
**Pull request by erwinmombay**
found some bugs in single pass for amp-ad-exit where we are dot access on a objects coming from a script configuration.

### CODE REVIEW

**Comment by erwinmombay** [comment]
ping @zhouyx for review

File: extensions/amp-ad-exit/0.1/amp-ad-exit.js [hunk 1]
```diff
@@ -127,13 +127,13 @@ export class AmpAdExit extends AMP.BaseElement {
       'CLICK_X': true,
       'CLICK_Y': true,
     };
-    if (target.vars) {
-      for (const customVarName in target.vars) {
+    if (target['vars']) {
```
Comment (zhouyx) [inline extensions/amp-ad-exit/0.1/amp-ad-exit.js]: Same question from the other PR. 
https://github.com/ampproject/amphtml/pull/19838/files/7753af78778728889a82ffc09a396493d62824d2#diff-0bf464ea83e306b02604a24025d949d1

target seems to be a locally used variable, that I believe we could let singlepass manipulate its field name. WDYT?

Comment (erwinmombay) [reply extensions/amp-ad-exit/0.1/amp-ad-exit.js]: I dont think thats the case here right? vars comes from the `config` object extracted from markup and JSON parsed.

Comment (zhouyx) [reply extensions/amp-ad-exit/0.1/amp-ad-exit.js]: I thought the `target` here comes from `this.target_[name]`, where comes from 
https://github.com/ampproject/amphtml/blob/5d09bae540bc116b0ddf1782db0a85d589ad4c3f/extensions/amp-ad-exit/0.1/amp-ad-exit.js#L273-L282
At which point, we've already read the `config` object.

**Pull request merged**